test/DvsaDocument/EntityTrait/CommonIdentityTraitTest.php

```diff
@@ -4,16 +4,24 @@
 
 use DvsaDocument\EntityTrait\CommonIdentityTrait;
 use PHPUnit\Framework\TestCase;
+use PHPUnit\Framework\MockObject\MockObject;
 
 /**
  * Test for CommonIdentityTrait
  */
 class CommonIdentityTraitTest extends TestCase
 {
+    /**
+     * @return void
+     *
+     * @psalm-suppress UndefinedDocblockClass
+     * The above is a trait not a class
+    */
     public function testTrait()
     {
-        /** @var CommonIdentityTrait $mock */
-        $mock = $this->getMockBuilder('DvsaDocument\EntityTrait\CommonIdentityTrait')->disableOriginalConstructor()->getMockForTrait();
+        // @phpstan-ignore-next-line
+        /** @var CommonIdentityTrait&MockObject $mock */
+        $mock = $this->getMockBuilder(CommonIdentityTrait::class)->disableOriginalConstructor()->getMockForTrait(); // @phpstan-ignore-line
```

Comment:
Same question, what is this suppressing?

Reply:
PHPStan complains `CommonIdentityTrait` is an undefined class (because it is a trait)